### ISSUE TITLE
Ability to reload the script/config? 

### ISSUE THREAD

**Issue by McKiller5252** [gtav-ent/GTAV-EnhancedNativeTrainer#94]
This would be an useful addition to the mod, Add a feature to reload the script/config so you don't have to restart GTA every time you've edited the config.

**Comment by gtav-ent** [comment]
Yes, quite possibly. Primarily just a case of finding room for it in the menus. Leave this open and I'll do it eventually.

In the meantime, **secret tip**: if you create a blank file called ScriptHookV.dev and put it in your GTA V directory, you can press Ctrl-R to get ScriptHook to reload the plugins. This does what you're asking.

**Comment by McKiller5252** [comment]
Cool thanks :)
